### DISCUSSION TITLE
上传文件报错405

### DISCUSSION THREAD

**Discussion by mashirozx** [AlistGo/alist#5078] [Q&A]
![Desktop Screenshot 2023 08 23 - 11 13 01 93](https://github.com/alist-org/alist/assets/16148054/a88991b4-44be-4bb0-845f-51033250e644)

我挂载的配置如上，上传小文件没问题，但是上传18Mb的图片时就报405了?

![Desktop Screenshot 2023 08 23 - 11 16 01 38](https://github.com/alist-org/alist/assets/16148054/2b960b07-7e75-4862-bcd7-f573de504243)

如果是我配置的问题请问应该如何排查呢？

**Comment by xhofe** [comment]
进入你要上传文件的文件夹。

**Reply by qingwan123** [reply]
我也有相同的问题，小文件可以，大文件就失败

**Reply by mashirozx** [reply]
在那个文件夹里面也一样，截图是我失误了

**Reply by qingwan123** [reply]
没错，都是一样的，上传大文件就会失败
